site/content/contribute/mobile/e2e/ios.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Setup and Run iOS E2E"
+date: 2020-09-01T09:00:00-00:00
+weight: 4
+subsection: Mobile End-to-End Testing
+---
+
+#### Local setup
+
+1. Install [applesimutils](https://github.com/wix/AppleSimulatorUtils):
+```
+brew tap wix/brew
+brew install applesimutils
+```
+2. Set XCode's build location so that the built app, especially debug, is expected at the project's location instead of the Library's folder which is unique/hashed.
+3. Open XCode, then go to **XCode > Preferences > Locations**.
+4. Under **Derived Data**, click **Advanced...**.
+5. Select **Custom > Relative to Workspace**, then set **Products** as **Build/Products**.
+6. Click **Done** to save the changes.
+
+#### Test run in debug mode
+
+This is the typical flow for local development and test writing:
+
+1. In one terminal window, run `make run` or `make run-ios` from the root folder.
```

Comment:
same as with Android.

Reply:
In iOS debug, we're building thru `react-native` not via [detox](https://github.com/mattermost/mattermost-mobile/blob/master/detox/.detoxrc.json#L5-L11). So either of these make commands are accurate and correct.

Reply:
No change yet as there's no equivalent `npm` script at the moment. Will update in the future once available.

Reply:
I'll make the changes to this doc later on. but it could've been replaced with `npx react-native run-ios` in the meantime.